DQM/SiStripMonitorClient/python/SiStripSourceConfigTier0_cff.py

```diff
@@ -71,6 +66,14 @@
     errorReplyDcs = cms.bool( True ),
 )
 
+from Configuration.StandardSequences.Eras import eras
+eras.stage2L1Trigger.toModify(SiStripMonitorClusterBPTX, 
+    BPTXfilter = dict(
+        stage2 = True,
```

Comment:
this parameter did not exist in the original. I suppose this is the reason for the errors.